FeedStoreChallenge/CoreDataCache+CoreDataClass.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  CoreDataCache+CoreDataClass.swift
+//  FeedStoreChallenge
+//
+//  Created by Dennis Nehrenheim on 14.05.21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+//
+
+import CoreData
+
+@objc(CoreDataCache)
+public class CoreDataCache: NSManagedObject {
```

Comment:
This class is an internal implementation detail of `CoreDataFeedStore` thus it should not be `public`
Also it is not going to be inherited from thus it should be `final`

Please make it `final` and `internal`

```suggestion
final class CoreDataCache: NSManagedObject {
```